### ISSUE TITLE
file download links broken

### ISSUE THREAD

**Issue by emmajhudgins** [mathewhauer/county_projections_official#5]
Hi Mathew,

Thanks for the great repo! I noticed the links to download the data are now broken because a 2017 release of the SEER data is available. I had to update script 002's download link to :
"https://seer.cancer.gov/popdata.thru.2016/yr1969_2016.19ages/us.1969_2016.19ages.adjusted.txt.gz"
and script 003's link to:
"https://seer.cancer.gov/popdata.thru.2016/yr1990_2016.19ages/us.1990_2016.19ages.adjusted.txt.gz" in order to get the download to work.

Emma

**Comment by DanielHadley** [comment]
Yes, we are stuck there as well. Thanks for adding this all open source!
